src/lib/vm-listener-hoc.jsx

```diff
@@ -60,9 +60,9 @@ const vmListenerHOC = function (WrappedComponent) {
                 this.props.vm.postIOData('userData', {username: this.props.username});
             }
 
-            // Re-request a targets update when the shouldEmitTargetsUpdate state changes to true
+            // Re-request a targets update when the shouldEmitUpdate state changes to true
```

Comment:
This one seems like it uncovered an error; I think it was introduced here: https://github.com/LLK/scratch-gui/commit/c799541d8ec5a80765392638f235ec52f811adc1

Does this need to be tested before we make this change?

Reply:
Yup, mentioned that in the PR
> The lint fixes were mostly just missing proptypes, not real bugs. But this includes a small bug where the target pane wouldn't automatically update if changes to the sprite occurred while in fullscreen/player mode (in vm-listener-hoc)
I did check manually that that issue was fixed.